src/test/java/uk/gov/hmcts/reform/prl/services/SendAndReplyServiceTest.java

```diff
@@ -1217,4 +1309,228 @@ public void testSendNotificationEmailOther() {
+                                           .messageAbout(MessageAboutEnum.APPLICATION)
+                                           .messageContent("some msg content")
+                                           .messageSubject("message subject")
+                                           .build()
+                    )
+                    .respondToMessage(YesOrNo.No)
+                    .messages(messages)
+                    .build())
+            .build();
+        Map<String, Object> dynamicData = getEmailDynamicData(caseData);
+        SendgridEmailConfig sendgridEmailConfig = SendgridEmailConfig.builder().toEmailAddress("testSolicitor@xyz.com")
+            .dynamicTemplateData(dynamicData)
+            .listOfAttachments(new ArrayList<>())
+            .languagePreference(LanguagePreference.english)
+            .build();
+        sendAndReplyService.sendNotificationToExternalParties(caseData, "authorisation");
+        verify(sendgridService).sendEmailUsingTemplateWithAttachments(
+            SendgridEmailTemplateNames.SEND_EMAIL_TO_EXTERNAL_PARTY,
+            "authorisation",
+            sendgridEmailConfig
+        );
+    }
+
+    @NotNull
+    private Map<String, Object> getEmailDynamicData(CaseData caseData) {
+        Map<String, Object> dynamicData = new HashMap<>();
+        dynamicData.put("caseReference", "12345");
+        dynamicData.put("dashBoardLink", manageCaseUrl + "/" + caseData.getId());
+        dynamicData.put("subject", "message subject");
+        dynamicData.put("content", "some msg content");
+        dynamicData.put("attachmentType", "pdf");
+        dynamicData.put("disposition", "attachment");
+        dynamicData.put("name", "Abc Xyz");
+        dynamicData.put("documentSize", 0);
+        dynamicData.put("messageAbout", "an application");
+        dynamicData.put("caseName", caseData.getApplicantCaseName());
+        return dynamicData;
+    }
+
+    @Test
+    public void testSendEmailNotificationToExternalPartiesWhenMessageIsNotExternal() throws IOException {
+        CaseData caseData = CaseData.builder().id(12345L)
+            .chooseSendOrReply(SendOrReply.SEND)
+            .caseTypeOfApplication("C100")
+            .replyMessageDynamicList(DynamicList.builder().build())
+            .applicants(null)
+            .respondents(null)
+            .sendOrReplyMessage(
+                SendOrReplyMessage.builder()
+                    .sendMessageObject(Message.builder()
+                                           .internalOrExternalMessage(InternalExternalMessageEnum.INTERNAL)
+                                           .externalMessageWhoToSendTo(null)
+                                           .messageAbout(MessageAboutEnum.APPLICATION)
+                                           .messageContent("some msg content")
+                                           .build())
+                    .respondToMessage(YesOrNo.No)
+                    .messages(messages)
```

Comment:
remove src/main/java/uk/gov/hmcts/reform/prl/services/SendAndReplyService.java from suppression list and check the coverage of the changes

Reply:
Addressed